grammar.y

```diff
@@ -966,37 +964,33 @@ function_call_generic:
   identifier '(' distinct_function_opt expr_list_opt ')' filter_opt
   {
     lowered := strings.ToLower(string($1))
-    if _, ok := AllowedFunctions[lowered]; !ok {
+    isCustom, ok := AllowedFunctions[lowered];
+    if !ok {
       yylex.(*Lexer).AddError(&ErrNoSuchFunction{FunctionName: string($1)})
     }
-    $$ = &FuncExpr{Name: Identifier(lowered), Distinct: $3, Args: $4, Filter: $6}
+
+    if isCustom {
```

Comment:
I make use of the flag to distinguish between a custom and a normal SQLite function

Reply:
The diff is a bit confusing. Try looking at the source code without diff. It's probably clearer